Validation: Added Missing Descriptions (#247)
MicrosoftDocs/bingmaps-docs: BingMaps/v8-web-control/map-control-concepts/configuration-driven-maps-framework/iframeable-configuration-map-example.md

---
title: "IFrameable Configuration Map Example | Microsoft Docs"

> description: "Describes the Iframeable configuration map and provides an example with HTML and provides an example to try a map configuration yourself."

Reviewer: ```suggestion
description: "Describes the Iframeable configuration map with an example in HTML enabling you to try a map configuration yourself."
```
